### DISCUSSION TITLE
[typecheck] Does it work with Vue 2?

### DISCUSSION THREAD

**Discussion by toomuchdesign** [vuedx/languagetools#230] [Q&A] [answered]
I'm trying to set up VueDX type checker on a legacy project written in Vue 2 and the check fails since it tries to parse components' `template` and `style` tags.

Since docs don't mention a specific Vue version, I've searched in GitHub for project using VueDX type checker and I found that all of them use Vue 3.

Is the type checker compatible with Vue 2? Or is there a way to make it work with a Vue 2 project?
Thanks so much 🙌

**Comment by znck** [comment, marked as answer]
No. I don't have bandwidth to support Vue 2.
